packages/react/src/components/ui2/typography/Text.tsx

```diff
@@ -6,7 +6,7 @@ export interface TextProps {
   level?: '1' | '2' | '3' | '4' | '5' | '6' | '7';
   fontFamilyVariant?: 'primary' | 'secondary';
   fontWeight?: 'normal' | 'bold';
-  textColorVariant?: 'primary' | 'secondary';
+  textColorVariant?: 'primary' | 'secondary' | 'tertiary';
```

Comment:
Do we really need a tertiary variant? (I understand that we have primary and secondary, but I have never seen tertiary => I think we should avoid this because it gets too complex). Why can we not just use secondary?